### DISCUSSION TITLE
Please do not post Support Questions/Feature Requests to Discussions

### DISCUSSION THREAD

**Discussion by Fenrirthviti** [obsproject/obs-studio#5206] [Announcements]
If you have a feature request, please submit it to the Ideas page: https://ideas.obsproject.com

If you have a support request, please use the forums or Discord: https://obsproject.com/help

We do not currently want to use GitHub Discussions for either of these, and any submissions will likely be either ignored or closed.

**Comment by roke-julian-lockhart** [comment]
@Fenrirthviti, what counts as a "Support" question / request (as opposed to a mere question)? I've yet to see this terminology utilised elsewhere, and Google isn't very assistive. Is it a synonym for bug report? I ask per [`#discussion-8526985`](https://github.com/obsproject/obs-studio/discussions/12333#discussion-8526985).
